src/constants/inventory-practice.js

```diff
@@ -0,0 +1,44 @@
+import {inventory} from "./inventory.js";
+
+// Opdracht 1a: Gebruik een array-methode om een array te maken met daarin alle tv-type namen. Log de uitkomst in de console.
+const tvTypes = inventory.map((tv)=> {
+    return tv.type;
+});
+console.log(tvTypes);
+
+// Opdracht 1b: Gebruik een array-methode om alle informatie te verzamelen van de tv's die volledig uitverkocht zijn. Log de uitkomst in de console.
+const soldOutTv = inventory.filter((tv) => {
+    let calculate = tv.originalStock - tv.sold;
+    return calculate === 0;
+});
+console.log(soldOutTv);
+
+// Opdracht 1c: Gebruik een array-methode om de informatie van het tv-type 'NH3216SMART' op te halen.
+const findTypeTv = inventory.find((tv) => {
+    return tv.type = 'NH3216SMART';
+});
+console.log(findTypeTv);
+
+// Opdracht 1d: Gebruik een array-methode om een lijst te maken van de merk- en tv-namen waarbij aangegeven wordt of deze geschikt zijn voor sport-kijkers (een verversingssnelheid van 100Hz of hoger).
+// Doe dit in het format { name: 'Toshiba HD TV', suitable: false }. Log de uitkomst in de console.
+const listSport = inventory.map((tv) => {
+    if (tv.refreshRate >= 100) {
+        return tv.brand + ' ' + tv.name +', suitable: true';
+    } else {
+        return tv.brand + ' ' + tv.name +', suitable: false';
+    }
```

Comment:
Het format klopt niet helemaal, en ik zou het ook net iets anders insteken om wat duplicatie te voorkomen
```suggestion
    return { name: tv.name, suitable: tv.refreshRate >= 100 };
```